experimental/iterators/lib/Conversion/IteratorsToLLVM/IteratorsToLLVM.cpp

```diff
@@ -196,6 +229,530 @@ struct PrintOpLowering : public ConversionPattern {
   }
 };
 
+//===----------------------------------------------------------------------===//
+// Helpers for creating Open/Next/Close functions
+//===----------------------------------------------------------------------===//
+
+/// Creates a new function at the parent module of originalOp with the given
+/// types and name and initializes the function body with a first block.
+FuncOp createOpenNextCloseFunc(Operation *originalOp, RewriterBase &rewriter,
```

Comment:
The name of this helper can be improved to something more descriptive, e.g. `createPrivateFuncInParentModule`.

Reply:
Sure! Done in next commit. I didn't make `private` part of the name but clarified that all `Open`/`Next`/`Close` functions are private in the function documentation.